src/Kiota.Builder/Lock/KiotaLock.cs

```diff
@@ -31,9 +32,9 @@ public class KiotaLock
     /// </summary>
     public string ClientClassName { get; set; } = string.Empty;
     /// <summary>
-    /// The class access modifier to use for the client classes.
+    /// The type access modifier to use for the client types.
     /// </summary>
-    public string? ClientClassAccessModifier { get; set; }
+    public string TypeAccessModifier { get; set; } = string.Empty;
```

Comment:
```suggestion
    public string TypeAccessModifier { get; set; } = "public";
```

I believe the default should be public here to support upgrade scenarios

Reply:
Implemented.